actionpack/lib/action_dispatch/http/content_security_policy.rb

```diff
@@ -345,7 +345,7 @@ def resolve_source(source, context)
             raise RuntimeError, "Missing context for the dynamic content security policy source: #{source.inspect}"
           else
             resolved = context.instance_exec(&source)
-            resolved.is_a?(Symbol) ? apply_mapping(resolved) : resolved
+            apply_mappings(Array.wrap(resolved))
```

Comment:
Should `require "active_support/core_ext/array/wrap"` be added in this file?

Reply:
Thanks, you are right!